frontend/packages/console-dynamic-plugin-sdk/upgrade-PatternFly.md

```diff
@@ -0,0 +1,59 @@
+# PatternFly Upgrade Notes
+
+A dynamic plugin targets one or more versions of OpenShift Console. Each Console version supports
+specific version(s) of [PatternFly](https://www.patternfly.org/) in terms of CSS styling.
+
+Check the [OpenShift Console Versions vs PatternFly Versions][console-pf-versions] table for
```

Comment:
Lets link the table here

Reply:
@jhadvig This is a [reference-style link](https://www.markdownguide.org/basic-syntax/#reference-style-links) to `console-pf-versions` declared at the end of the Markdown document.

```md
[console-pf-versions]: ./README.md#openshift-console-versions-vs-patternfly-versions
```